observability/tools/loki.md

```diff
@@ -0,0 +1,39 @@
+# Loki
+
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system, created by Grafana
+Labs inspired by the learnings from Prometheus. Loki is commonly referred as Prometheus but for logs, which
+makes total sense once both tools follow the same architecture, which is a agent collecting metrics in each
+of the components of the software system, a server which stores the logs and also the Grafana dashboard, which
+access the loki server to build its visualizations and queries. That being said, Loki has three main
+components:
+
+## Promtail
+
+Its the agent portion of Loki. It can be used to grab metrics from several different places, like var/log/ for
+example. The configuration of the Promtail is a yaml file called ```config-promtail.yml``` commonly store on
+the  ```/usr/local/bin``` path. In this file, its described all the paths and log sources that will be
+aggregated on Loki Server.
+
+## Loki Server
+
+Responsible for receiving and storing all the logs received from all the different systems. The Loki is also
+responsible for the queries done on Grafana, for example.
+
+## Grafana Dashboards
+
+Grafana Dashboards are responsible for creating the visualizations and performing queries. After all, it will
+be a web page that people with the right access can log into to see, query and create alerts for the aggregated
+logs.
+
+## Why use Loki
+
+The main reason to use Loki instead of other log aggregation tools, is that Loki optimizes the necessary
+storage. It does that by following the same pattern as prometheus, which index the labels and make chunks
+of the log itself, using less space than just storing the raw logs.
+
+## References
+
+- [Loki Official Site](https://grafana.com/oss/loki/)
```

Comment:
https://grafana.com/docs/loki/latest/best-practices/current-best-practices/

This is also a good link to add

Reply:
nice link, just added it!